.github/workflows/container_build_fix.yml

```diff
@@ -0,0 +1,36 @@
+# this workflow can be triggered manually to initiate any build if it fails
+name: fix container build
+
+"on":
+  workflow_dispatch:
+    inputs:
+      container_name:
+        type: choice
+        required: true
+        description: "select image name to build"
+        options: [universal]
+        default: "universal"
+      container_tags:
+        type: string
+        required: false
+        description: "container image tags"
+        default: "fix"  # this is a special tag that can be used for testing
```

Comment:
Let's find something more saying that fix. Maybe "test" or "adhoc" or "manual-build"